components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/group/SCIMGroupHandlerTest.java

```diff
@@ -73,16 +72,32 @@ public class SCIMGroupHandlerTest extends PowerMockTestCase {
     @Mock
     private PreparedStatement mockedPreparedStatement;
 
+    private MockedStatic<SCIMCommonUtils> scimCommonUtils;
+    private MockedStatic<IdentityDatabaseUtil> identityDatabaseUtil;
+    private MockedStatic<IdentityTenantUtil> identityTenantUtil;
+    private MockedStatic<StringUtils> stringUtils;
+
     @BeforeMethod
-    public void setUp() throws Exception {
+    public void setUp() {
+
         initMocks(this);
+        scimCommonUtils = mockStatic(SCIMCommonUtils.class);
+        identityDatabaseUtil = mockStatic(IdentityDatabaseUtil.class);
+        identityTenantUtil = mockStatic(IdentityTenantUtil.class);
+        stringUtils = mockStatic(StringUtils.class);
```

Comment:
Mocking the `StringUtils` seems to be little bit suspicious. There seems to be no mocks added as well? What is the requirement of mocking `StringUtils`?